src/index.html

```diff
@@ -11,7 +11,220 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="main-form"
+    >
+
+      <fieldset class="block-forms personal-block">
+        <legend class="title">
+          Personal information
+        </legend>
+        <div class="personal-inf">
```

Comment:
One letter won't make the class name too long)
```suggestion
        <div class="personal-info">
```